src/Symfony/Component/Mailer/Bridge/Mailjet/Transport/MailjetApiTransport.php

```diff
@@ -126,6 +138,11 @@ private function getPayload(Email $email, Envelope $envelope): array
         }
 
         foreach ($email->getHeaders()->all() as $header) {
+            if (\array_key_exists($header->getName(), self::HEADER_TO_MESSAGE)) {
+                $convertConf = self::HEADER_TO_MESSAGE[$header->getName()];
+                $message[$convertConf[0]] = $this->castCustomHeader($header->getBodyAsString(), $convertConf[1] ?? 'string');
```

Comment:
The `$header->getValue()` funcion does not work to get stringified value ?

Reply:
`Symfony\Component\Mime\Header\HeaderInterface` does not have `getValue()` method. Do you mean `toString()` method?

Reply:
sorry, I may have been mistaken then